rest_framework/fields.py

```diff
@@ -778,6 +778,8 @@ def to_internal_value(self, data):
         return data
 
     def to_representation(self, value):
+        if value is None:
+            return None
```

Comment:
The `None` case is handled as part of the regular serialization machinery - shouldn't need to be handling them on a per-field basis. (Tho perhaps something else wonky?)